src/Umbraco.Web/Security/MembershipHelper.cs

```diff
@@ -736,6 +751,9 @@ public virtual Attempt<PasswordChangedModel> ChangePassword(string username, Cha
                         username,
                         membershipProvider.RequiresQuestionAndAnswer ? passwordModel.Answer : null);
 
+                    if (membershipProvider.IsUmbracoUsersProvider() && backofficeUserManager != null && userId >= 0)
+                        backofficeUserManager.RaisePasswordResetEvent(userId);
```

Comment:
This isn't a great place to raise this event since this method deals with both users and members and now there's a lot of extract checks in here specifically for raising user events. These events should be raised where the code uses this method.